<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HellingerNewSegmentWarning</class>
 <widget class="QDialog" name="HellingerNewSegmentWarning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Segment Warning</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_warning_text">
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Warning text</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radio_add">
       <property name="text">
        <string>add to existing segment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_replace">
       <property name="text">
        <string>replace existing segment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_insert">
       <property name="text">
        <string>insert as new segment, re-numbering the following segments</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="button_cancel">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="button_ok">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>HellingerNewSegmentWarning</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_ok</sender>
   <signal>clicked()</signal>
   <receiver>HellingerNewSegmentWarning</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
